### DISCUSSION TITLE
Troubles with Proof Verify

### DISCUSSION THREAD

**Discussion by tobycm** [community/community#36845] [GitHub Education]
Hi GitHub Community!

I am applying for GitHub Global Campus but I have some troubles in the way: 
![image](https://user-images.githubusercontent.com/62174797/197078425-b160f860-c289-4757-a108-d47076aaef76.png)

This is my proof (type is dated school ID) (with no censors):

![student_idv3 - Copy](https://user-images.githubusercontent.com/62174797/197078937-0ee436a1-eae1-4ea7-97c0-64cd6375aa91.jpg)

I've already tried to reach out to GitHub Support but...
![gituh support](https://user-images.githubusercontent.com/62174797/197079056-1c4ed22e-4551-4235-974d-541c44ea500f.png)

Thanks in advance.
`Exit code 0`

**Comment by ossanna16** [comment]
Thank you for asking a good question @tobycm, I'm glad you found our community 🙂
